website/templates/include/profile/schools.mako

```diff
@@ -96,7 +95,7 @@
             </div>
 
             <div>
-                <a class="btn btn-default" data-bind="click: addContent">
+                <a class="btn btn-default" data-bind="css: enablebtn, click: addContent">
```

Comment:
I initially tried to use disable, and was not able to get it to work with our new styleguide class.  I might have not implemented it correctly, but I can put some more time into it if you like.